werkzeug/exceptions.py

```diff
@@ -520,6 +520,20 @@ class Locked(HTTPException):
     )
 
 
+class FailedDependency(HTTPException):
+
+    """*424* `Failed Dependency`
+
+    Used if the method could not be performed on the resource 
```

Comment:
There is a trailing whitespace after `resource`, which causes the travis-ci failure.

Reply:
Fixed, but still failing